### PR TITLE
Add a command to open an ssh tunnel to the specified farms DB

### PR DESCRIPTION
**Pull request by BobTheBotMaker**
Once the tunnel has been established you connect to the db in one of two ways.

psql postgres://USERNAME:PASSWORD@localhost:5433/DATABASENAME

Or using pgadmin with the above parameters.

NOTE THE PORT NUMBER CHANGE. This is because you may have postgres running locally.

### CODE REVIEW

File: bin/ttmscalr [hunk 1]
```diff
@@ -1201,4 +1202,33 @@ Main {
       url_shard(replacement)
     end
   end
+
+  mode 'psql:tunnel' do
+    description 'Open an SSH tunnel to an RDS DB'
```
Comment (TheJefe) [inline bin/ttmscalr]: This might be a bit confusing.  It's not really opening it to an RDS DB, it i opening it to a debug role on a farm

**Comment by TheJefe** [comment]
Other then the command description being a little confusing, this looks good

File: bin/ttmscalr (hunk 1 above)
Comment (BobTheBotMaker) [inline bin/ttmscalr]: No, it actually opens the tunnel to the DB Server for the farm, it just routes it through the debug role. If you look in ssher.rb line 32 you can see where I build the tunnel spec. The only thing available on the tunnel is the Postgres server. Ill update the wording a bit though.

**Comment by joelmccracken** [comment]
I just tried this -- worked great for me. 

One thing to note, the tunnel command will need to be run in a separate shell, as the command will need to still be running while you use the tunnel.

**Pull request merged**